src/Fantomas.Core/SyntaxOak.fs

```diff
@@ -1738,10 +1738,6 @@ type Expr =
             match node.CopyInfo with
             | Some _ -> false
             | None -> true
-        | Expr.NamedComputation node ->
```

Comment:
Just a thought but I think this member should be extracted into a function in `Context`.
If you have that, you can pass the `FormatConfig` into it and just add an extra `when` expression for the named computation expression.

I think this makes sense as it further removes the coupling between the `Oak` model and any coding style.

By refactoring this, I think the current flow could be preserved slightly easier.